graphics/include/ignition/common/ColladaExporter.hh

```diff
@@ -47,6 +49,10 @@ namespace ignition
       public: virtual void Export(const Mesh *_mesh,
           const std::string &_filename, bool _exportTextures = false);
 
+      public: virtual void Export(const Mesh *_mesh,
+          const std::string &_filename, bool _exportTextures,
+          const std::vector<math::Matrix4d> &_submeshToMatrix);
```

Comment:
I forget the ABI rules. Is it ok to add a virtual function if it's at the end of the class declaration?

Reply:
I believe we haven't been adding new virtual functions, even at the end.

I don't think we have any classes subclassing `ColladaExporter`, so I think this could be non-virtual.